geomesa-kafka/geomesa-kafka-confluent/src/main/scala/org/locationtech/geomesa/kafka/confluent/ConfluentKafkaDataStoreFactory.scala

```diff
@@ -54,14 +62,50 @@ object ConfluentKafkaDataStoreFactory extends GeoMesaDataStoreInfo with LazyLogg
   override val DisplayName = "Confluent Kafka (GeoMesa)"
   override val Description = "Confluent Apache Kafka\u2122 distributed log"
 
-  val SchemaRegistryUrl = new GeoMesaParam[URL]("kafka.schema.registry.url", "URL to a confluent schema registry server, used to read Confluent schemas (experimental)")
+  val SchemaRegistryUrl =
+    new GeoMesaParam[URL](
+      "kafka.schema.registry.url",
+      "URL to a confluent schema registry server, used to read Confluent schemas (experimental)",
+      optional = false
```

Comment:
if you have schema overrides, then you don't need a schema registry, correct? in the docs it said one use case is to not have a schema registry, but here we are requiring one.

Reply:
it would probably be easiest to just update the docs to remove that reference, since we use this param to load the confluent KDS instead of the regular one.